client/src/Components/FormComboBox.js

```diff
@@ -70,16 +69,20 @@ function FormComboBox({ label, property, required, onChange, options, showRequir
     input.value = value;
   }, [value]);
 
+  const hasError =
+    (validationState === ValidationState.REQUIRED_ERROR || validationState === ValidationState.RANGE_ERROR) &&
+    ((focused && value?.length > 1) || !focused);
+
   useEffect(() => {
     const { current } = ref;
     // manually add/remove validation classes to the custom input added by USWDS
     const input = current.querySelector('input[type="text"]');
-    if (validationState === ValidationState.ERROR) {
+    if (hasError && !focused) {
```

Comment:
Does `focused` need to be checked here, if `hasError` is based on whether it's focused?

Reply:
resolved in most recent push